src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -1053,15 +1069,24 @@ internal virtual void SaveLastDebuggerAgentBufferReceivedToContext(SessionId ses
         {
         }
 
-        internal async Task<bool> OnReceiveDebuggerAgentEvent(SessionId sessionId, JObject args, CancellationToken token)
+        internal async Task<Result> GetLastDebuggerAgentBuffer(SessionId sessionId, JObject args, CancellationToken token)
```

Comment:
Will the buffer be relevant once `OnReceiveDebuggerAgentEvent` is done? If not, then this method should be merged with that.

Reply:
For firefox yes.